packages/sanity/src/core/releases/components/dialog/ReleaseDetailsDialog.tsx

```diff
@@ -40,6 +40,7 @@ export function ReleaseDetailsDialog(props: ReleaseDetailsDialogProps): JSX.Elem
       hue: bundle?.hue || 'gray',
       icon: bundle?.icon || 'cube',
       publishedAt: bundle?.publishedAt,
+      releaseType: bundle?.releaseType || 'asap',
```

Comment:
Maybe create a const for default type so it's defined in one place?
e.g.
`const DEFAULT_RELEASE_TYPE = 'asap'`

Reply:
Good point :) 
Added!